blindsign/blindrsa/brsa_test.go

```diff
@@ -463,3 +494,48 @@ func BenchmarkBRSA(b *testing.B) {
 		b.Fatal(err)
 	}
 }
+
+func Example_blindrsa() {
+	// Setup (offline)
+
+	// Server: generate an RSA keypair.
+	sk, err := rsa.GenerateKey(rand.Reader, 2048)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to generate RSA key: %v", err)
+		return
+	}
+	pk := &sk.PublicKey
+	server := NewSigner(sk)
+
+	// Client: stores Server's public key.
+	verifier := NewVerifier(pk, crypto.SHA384)
+
+	// Protocol (online)
+
+	// Client blinds a message.
+	msg := []byte("alice and bob")
```

Comment:
interestingly, having simpler test vectors like this in the spec could be beneficial. It's not going to be for this protocol as it's already RFC, but I sense it would be quite useful to have a test vector that fits in a simple example.

Reply:
a good suggestion for those spec who hasn't been published (frozen) yet.